spec/factories/users.rb

```diff
@@ -1,6 +1,14 @@
 FactoryGirl.define do
   factory :user do
-    sequence(:email_address, User.count) {|n| "liono#{n}@some-cartoon-show.com" }
+    # don't freak out if the database isn't created yet
+    # http://stackoverflow.com/a/27745684/358804
```

Comment:
This seems a little hacky. If it's needed, should it be present in the `rails_helper.rb` instead?